products/opensuse/main.pm

```diff
@@ -776,15 +744,16 @@ elsif (get_var("WICKED")) {
     load_wicked_tests();
 }
 elsif (get_var("REGRESSION")) {
-    if (get_var("KEEPHDDS")) {
-        load_login_tests();
-    }
-    else {
+    if (check_var("REGRESSION", "installation")) {
+        load_boot_tests();
         load_inst_tests();
         load_reboot_tests();
+        loadtest "x11regressions/x11regressions_setup";
+        loadtest "console/hostname";
+        loadtest "console/force_cron_run";
+        loadtest "shutdown/grub_set_bootargs";
+        loadtest "shutdown/shutdown";
     }
-
-    load_x11regresion_tests();
```

Comment:
Subroutine load_x11regresion_tests was dropped because

1. It's not enabled in any testsuite of openqa.opensuse.org, and cases within the subroutine will be re-enabled in my later work.
2. This subroutine is too old, three of the cases were already deprecated:
- "x11regressions/gnomecase/Gnomecutfile";
- "x11regressions/pidgin/pidgin_googletalk";
- "x11regressions/pidgin/pidgin_msn";